src/seq_mv/csr_matop.c

```diff
@@ -128,6 +128,114 @@ hypre_CSRMatrixAdd( hypre_CSRMatrix *A,
    return C;
 }       
 
+/*--------------------------------------------------------------------------
+ * hypre_CSRMatrixBigAdd:
+ * adds two CSR Matrices A and B and returns a CSR Matrix C;
+ * Note: The routine does not check for 0-elements which might be generated
+ *       through cancellation of elements in A and B or already contained
+ in A and B. To remove those, use hypre_CSRMatrixDeleteZeros 
+ *--------------------------------------------------------------------------*/
+
+hypre_CSRMatrix *
+hypre_CSRMatrixBigAdd( hypre_CSRMatrix *A,
```

Comment:
Why is Add is so special that we need a Big version of it separately? Just curious